src/coreclr/vm/memberload.cpp

```diff
@@ -1129,11 +1129,7 @@ MemberLoader::FindMethod(
         }
```

Comment:
Also delete

```
   SString targetName(SString::Utf8Literal, pszName);
    ULONG targetNameHash = targetName.HashCaseInsensitive();
```

at the top of the start of the method. (Multiple places in this file.)

Reply:
Doh. I updated that for testing but forgot to remove it.